sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/ApproximatePercentile.scala

```diff
@@ -245,7 +245,7 @@ object ApproximatePercentile {
         val result = new Array[Double](percentages.length)
         var i = 0
         while (i < percentages.length) {
-          result(i) = summaries.query(percentages(i))
+          result(i) = summaries.query(percentages(i)).get
```

Comment:
Yes. I think it is impossible to hit it. We need some test cases to ensure it. See my next comment.